torch/utils/cpp_extension.py

```diff
@@ -0,0 +1,60 @@
+import os.path
+import re
+import subprocess
+import warnings
+from setuptools.command.build_ext import build_ext
+
+MINIMUM_GCC_VERSION = (4, 9)
+ABI_INCOMPATIBILITY_WARNING = '''
+Your compiler ({}) may be ABI-incompatible with PyTorch.
+Please use a compiler that is ABI-compatible with GCC 4.9 and above.
+See https://gcc.gnu.org/onlinedocs/libstdc++/manual/abi.html.'''
+
+
+def check_compiler_abi_compatibility(compiler):
```

Comment:
@soumith @zdevito

Reply:
That should be good enough for now. We should disable the extension test on windows builds. It is not clear to me that the osx build failures are related, but that is the only other test failing.

Reply:
How are we testing this?